manuskript.spec

```diff
@@ -1,4 +1,15 @@
 # -*- mode: python -*-
+import distutils.util
+
+COMPILING_PLATFORM = distutils.util.get_platform()
+if COMPILING_PLATFORM == 'win-amd64':
+    platform = 'win'
+    STRIP = False
+elif COMPILING_PLATFORM == 'linux-x86_64':
+    platform = 'nix64'
+    STRIP = True
```

Comment:
I found some generic code to do platform detection and thought it would be useful to add this if anyone wants to use it in the future. I'm happy to remove the other platforms that aren't needed.

Reply:
In my opinion code that isn't used will unnecessarily add to complexity.  Further since the check for macOS seems to only be used to build an app bundle on macOS, it makes sense to place this check at that section of the code.